### PR TITLE
Update duties_agreement.md

### PR DESCRIPTION
**Pull request by karenword**
proposed changes to the Trainer Agreement for 2023

### CODE REVIEW

File: topic_folders/instructor_training/duties_agreement.md [hunk 1]
```diff
@@ -12,31 +12,39 @@ discussions about Instructor training. Trainers have opportunities to influence
...
-###### Participation Requirements
-Participation requirements for automatic renewal are at least:
+In the case of option A, above, certification will be renewed automatically when records indicate that Trainers have completed all activities expected 
+for Active Trainers, detailed below. If records are incomplete, Trainers may self-report their activities via a form. The Trainers Leadership Committee 
+will review form responses and recommend either renewal or a meeting (option B) based on reported activities.
+
+###### Participation Expectations for Active Instructor Trainers
+In order to be renewed based on prior participation, a Trainer is expected to participate as follows during the 1 year period prior to renewal:
+
+- teach 1 Instructor Training event, and
```
Comment (jcoliver) [inline topic_folders/instructor_training/duties_agreement.md]: If we want to be pedantic, we could add "at least" before each of these counts ("teach _at least_ 1 Instructor Training event", etc.)

File: topic_folders/instructor_training/duties_agreement.md [hunk 2]
```diff
@@ -12,31 +12,39 @@ discussions about Instructor training. Trainers have opportunities to influence
 community, through leadership roles. Trainers often play leadership roles in their home communities, mentoring local Instructors, and modeling best practices.
 
 
-##### Certification Renewal Process
+##### Active Status Renewal Process
 
-Version 1.0.0 -- Approved 16 February, 2022
+Version 1.0.0 -- Draft
```
Comment (jcoliver) [inline topic_folders/instructor_training/duties_agreement.md]: Increment version number to 1.0.1?

**Review by jcoliver — COMMENTED** [review]
Looks good - two minor comments.

File: topic_folders/instructor_training/duties_agreement.md (hunk 1 above)
Comment (brownsarahm) [reply topic_folders/instructor_training/duties_agreement.md]: I agree on stating minumum

File: topic_folders/instructor_training/duties_agreement.md (hunk 2 above)
Comment (karenword) [inline topic_folders/instructor_training/duties_agreement.md]:
```suggestion
Version 1.0.1 -- Draft
```

File: topic_folders/instructor_training/duties_agreement.md (hunk 1 above)
Comment (karenword) [inline topic_folders/instructor_training/duties_agreement.md]:
```suggestion
- teach at least 1 Instructor Training event, and  
```

File: topic_folders/instructor_training/duties_agreement.md [hunk 3]
```diff
@@ -12,31 +12,39 @@ discussions about Instructor training. Trainers have opportunities to influence
...
+In the case of option A, above, certification will be renewed automatically when records indicate that Trainers have completed all activities expected 
+for Active Trainers, detailed below. If records are incomplete, Trainers may self-report their activities via a form. The Trainers Leadership Committee 
+will review form responses and recommend either renewal or a meeting (option B) based on reported activities.
+
+###### Participation Expectations for Active Instructor Trainers
+In order to be renewed based on prior participation, a Trainer is expected to participate as follows during the 1 year period prior to renewal:
+
+- teach 1 Instructor Training event, and  
+- host 2 teaching demos, and  
+- attend 4 Trainer community meetings
```
Comment (karenword) [inline topic_folders/instructor_training/duties_agreement.md]:
```suggestion
- host at least 2 teaching demos, and  
- attend at least 4 Trainer community meetings  
```

**Pull request merged**